extensions/amp-story-auto-ads/0.1/amp-story-auto-ads.js

```diff
@@ -200,6 +200,13 @@ export class AmpStoryAutoAds extends AMP.BaseElement {
             }
           });
         }
+      })
+      .catch((e) => {
+        dev().expectedError(
```

Comment:
(Just driving by) Would it work to guard this with `viewer.hasCapability('playerExperiments')` instead of catching the error?

Reply:
👋 Yeah that would be better, unfortunately the player does not support the cap param for this feature yet. We have the request out and this todo
https://github.com/ampproject/amphtml/blob/cf62ae1d6a12b70dfb6a7bd8ce66ed5bc6e30d1b/extensions/amp-story-auto-ads/0.1/amp-story-auto-ads.js#L113